projects/components/src/stepper/stepper.component.test.ts

```diff
@@ -0,0 +1 @@
+describe('Stepper Component', () => {});
```

Comment:
You would need to add at least one spec here.

Reply:
added tests.